assets/airports/efhk.json

```diff
@@ -0,0 +1,1361 @@
+            "draw": []
+        },
+        "NAPUN": {
+            "icao": "NAPUN",
+            "name": "NAPUN",
+            "suffix": {"04L":"1B"},
+            "body": [["NAPUN", "A70+|S250-"], "HK890", ["HK896", "A30+"], "HK849", ["HK851", "A23|S220-"], ["HK852", "A23"], ["PODOM", "A23|S210-"]],
+            "draw": []
+        },
+        "PEXEN": {
+            "icao": "PEXEN",
+            "name": "PEXEN",
+            "suffix": {"04L":"4B", "22R":"3V"},
+            "entryPoints": {
+                "PEXEN": [["PEXEN", "A70+|S250-"]]
+            },
+            "body": ["KUDOP"],
+            "rwy": {
+                "04L": [["HK874", "A30+|S220-"], ["HK873", "A30+"], ["HK851", "A23|S220-"], ["HK852", "A23"], ["PODOM", "A23|S210-"]],
+                "22R": [["HK886", "A30+"], ["ADBIX", "A20+|S220-"], ["HK887", "A20|S220-"], "HK883", ["TEDMU", "A20|S210-"]]
+            },
+            "draw": []
+        },
+        "ROPAM": {
+            "icao": "ROPAM",
+            "name": "ROPAM",
+            "suffix": {"04L":"1B", "04R":"1R", "15":"1M", "22L":"2A", "22R":"2V", "33":"1W"},
+            "entryPoints": {
+                "LEDUN": ["LEDUN"]
+            },
+            "body": [["ROPAM", "S250-"]],
+            "rwy": {
+                "04L": ["HK711", ["HK804", "A30+|S220-"], "HK801", ["PODOM", "A23+|S210-"]],
+                "04R": ["HK711", ["HK894", "A33+|S220-"], ["HK814", "A33+|S220-"], "HK811", ["GEDLO", "A33+|S210-"]],
+                "15" : [["HK817", "A30+"], ["SUTAX", "A20+|S220-"], ["HK822", "A20+|S220-"], "HK821", ["ELKUN", "A20+|S210-"]],
+                "22L": [["HK906", "A30+|S220-"], "HK904", ["BIMVU", "A30+|S210-"]],
+                "22R": [["HK881", "A30+|S220-"], "HK883", ["TEDMU", "A20|S210-"]],
+                "33" : [["HK844", "A30+|S220-"], "HK861", ["OBONU", "A30+|S210-"]]
+            },
+            "draw": []
+        },
+        "VEPIN": {
+            "icao": "VEPIN",
+            "name": "VEPIN",
+            "suffix": {"04L":"1B", "04R":"1R", "15":"1M", "22L":"2A", "22R":"2V", "33":"1W"},
+            "entryPoints": {
+                "ARLOM": ["ARLOM"],
+                "ROKVI": ["ROKVI"]
+            },
+            "body": [["VEPIN", "A100+"], ["MAMOP", "A70+|S250-"]],
+            "rwy": {
+                "04L": ["HK807", "HK711", ["HK804", "A30+|S220-"], "HK801", ["PODOM", "A23+|S210-"]],
+                "04R": ["HK807", "HK711", ["HK894", "A33+|S220-"], ["HK814", "A33+|S220-"], "HK811", ["GEDLO", "A33+|S210-"]],
+                "15" : [["HK888", "A30+"], ["HK822", "A20+|S220-"], "HK821", ["ELKUN", "A20+|S210-"]],
+                "22L": [["HK892", "A30+|S220-"], "HK904", ["BIMVU", "A30+|S210-"]],
+                "22R": [["HK893", "A30+|S220-"], "HK883", ["TEDMU", "A20|S210-"]],
+                "33" : [["HK844", "A30+|S220-"], "HK861", ["OBONU", "A30+|S210-"]]
+            },
+            "draw": []
+        }
+    },
+
+    "spawnPatterns": [
```

Comment:
Game: 58 ac/h
Wikipedia: 18 ac/h

The acph in game should be reduced as it exceeds 3x hourly operations of the real life Vantaa airport.